mqtt2notifysend.py

```diff
@@ -4,23 +4,36 @@
 import subprocess
 import paho.mqtt.client as mqtt
 
-BROKER = "192.168.0.90"
-TOPIC_SUB = "dev/pc/toast"
-TOPIC_STAT = "dev/pc/stat"
-PORT = 1883
+# The CLIENT_NAME must be unique on the broker.
+# USERNAME, PASSWORD and 8883 port are for secure connection with the Broker.
+# If you do not use SSL/TLS connection, change the port to 1883.
+
+BROKER = "192.168.X.XXX"
+CLIENT_NAME = "your_computer_name"
+USERNAME = "your_username"
+PASSWORD = "ypur_password"
+CLEAN_SESSION = True
+USER_DATA = None
+PROTOCOL = mqtt.MQTTv311
+TOPIC_SUB = "dev/"+CLIENT_NAME+"/toast"
+TOPIC_STAT = "dev/"+CLIENT_NAME+"/stat"
+PORT = 8883
 KEEPALIVE = 60
-PAYLOAD_ON = "Client Online"
-PAYLOAD_OFF = "Client Offline"
+PAYLOAD_ON = "Online"
+PAYLOAD_OFF = "Offline"
 DELIMITER = ";;;"
 DEFAULT_TITLE = "MQTT"
 
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 ICON = os.path.join(CURRENT_DIR, "icon.png")
 
 
-def on_connect(client: mqtt.Client, userdata, flags, rc):
-    client.subscribe(TOPIC_SUB)
-    client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+def on_connect(mqtt_client: mqtt.Client, userdata, flags, rc):
+    mqtt_client.subscribe(TOPIC_SUB)
+    mqtt_client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+    mqttClient.will_set(TOPIC_STAT, PAYLOAD_OFF, retain=True)
+    # Comment the following line if you do not use SSL/TLS connection with the Broker
```

Comment:
USERNAME and PASSWORD variables can be empty strings or None if the user won't want to use authentication